code/game/machinery/computer/communications.dm

```diff
@@ -769,8 +769,14 @@
 		if("message")
 			status_signal.data["top_text"] = data1
 			status_signal.data["bottom_text"] = data2
+			message_admins("[key_name(usr)][ADMIN_LOOKUPFLW(usr)][ADMIN_SMITE(usr)] has changed the station status display message to \"[data1] [data2]\" [loc_name(usr)]")
```

Comment:
This doesn't feel urgent enough to spam admins as opposed to the things that so (atmos, bombs, etc.)